docs/docs-components/COMPONENT_DATA.js

```diff
@@ -672,6 +672,7 @@ const GENERAL_COMPONENT_LIST: $ReadOnlyArray<ListItemType> = [
       figma: null,
     },
   },
+
```

Comment:
Committing little nonsense changes like this makes our git history dirtier, in that it will look like this PR makes changes to this file, but it doesn't really. Please revert this change for a cleaner git history. (This matters when Future Us are looking back at changes)

Reply:
OK, sorry. will revert.

Reply:
did my revert work?

Reply:
Yep! Thanks for that 🙂